docs/docsite/rst/user_guide/become.rst

```diff
@@ -355,19 +355,21 @@ the user token, it will continue to use the limited token during execution.
 
 Before Ansible 2.5, a token was only able to be elevated when UAC was disabled
 or the remote user had the ``SeTcbPrivilege`` assigned. This restriction has
-been lifted in Ansible 2.5 and a user that is a member of the
-``BUILTIN\Administrators`` group should have an elevated token during the
-module execution.
+been lifted in Ansible 2.5 and the remote user only requires the
```

Comment:
If you split out the 2.5-and-forward docs changes, like this section, into a separate PR, it will be easier to backport the right set of changes to 2.7, 2.6, and 2.5.

Reply:
Moved all the stuff that isn't related to this new feature here https://github.com/ansible/ansible/pull/48149